scripts/release-test-script-ut-control.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
```

Comment:
same think big picture need a list of what the goals are of the scripts.

So we can review and see gaps in an easy table,, can be part of this file, or a separate doc if you prefer.. A markdown file maybe ideal because you can add pictures though.

Reply:
Done